BranchMetrics/ios-branch-deep-linking-attribution
# SDK - 1758 - Added gbraid_timestamp on v1/open and v2/event requests.

## Reference
SDK-1758 -- Send 'gbraid_timestamp' for v1/open and v2/event

## Summary
Added parameter `'gbraid_timestamp'` in v1/open and v2/event  requests.

## Type Of Change

- [ X] New feature (non-breaking change which adds functionality)

## Testing Instructions
* URL for deep linking - Add param gbraid in URL  like - https://bnctestbed.app.link/?gbraid=“123445” .
Launch test bed by clicking above URL. Check logs. Logs of v1/open and v2/event requests should contain parameter 
gbraid_timestamp.
* All unit tests should pass.









cc @BranchMetrics/saas-sdk-devs for visibility.
